creator-node/src/utils/requestRange.ts

```diff
@@ -45,7 +45,7 @@ export const getRequestRange = (req: Request) => {
  */
 export const formatContentRange = (
   start: number,
-  end: number,
+  end: number | undefined,
```

Comment:
nit: don't need to change but in the future I'd prefer to just do `end?: number`. maybe there's a linter rule for this?

Reply:
Very minor difference, `end?: number` means the param is optional, `end: number | undefined` means the param can be a number of undefined. Usually those two statements are the same but unfortunately `end` is not the last param to the function. Only parameters at the end can be optional

tl;dr
This works for `end: number | undefined` but not for `end?: number`
```js
formatContentRange(0, undefined, 10)
```

Reply:
oh this is a function not a type, right. ok cool cool thanks for clarifying 👍